package.json

```diff
@@ -14,8 +14,8 @@
     "build_test": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && jasmine",
     "build_cover": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && istanbul cover jasmine",
     "build_docs": "./docgen.sh",
-    "lint_perf": "eslint perf/",
-    "lint_spec": "eslint spec/**/*.js",
+    "lint_perf": "eslint perf/ --fix",
+    "lint_spec": "eslint spec/**/*.js --fix",
```

Comment:
now let machine handle hard work, let people do easy part of writing code ;)

--fix option automatically corrects some of minor issues (spacing, etcs) but still some of lint error's won't be fixed. tslint does not have same options yet.